test/unit/operatoramp.spec.js

```diff
@@ -38,88 +38,37 @@ describe('OperatorAmp', () => {
       };
 
       const amp = new OperatorAmp(sampleAmp);
-      assert.equal(JSON.stringify(amp), JSON.stringify({
-        xp: 5458755,
-        polarized: 0,
-        amp: {
-          itemName: 'META',
+      assert.equal(JSON.stringify(amp),
+        JSON.stringify({
           xp: 5458755,
           polarized: 0,
-          features: 8,
-          pricol: {
-            t0: '5B73D6',
-            t1: '4559A5',
-            t2: '33589B',
-            t3: '6F7DB3',
-            m0: '33589B',
-            m1: '5E71B3',
-            en: '33589B',
-            e1: '5E71B3',
-          },
-          uniqueName: '/Lotus/Weapons/Sentients/OperatorAmplifiers/OperatorAmpWeapon',
-          upgrades: [
-            {
-              uniqueName: '/Lotus/Upgrades/CosmeticEnhancers/OperatorAmps/IncreasedCriticalDamageOnCriticalStrike',
-              rank: 3,
+          amp: {
+            itemName: 'META',
+            xp: 5458755,
+            polarized: 0,
+            features: 8,
+            pricol: {
+              t0: '5B73D6', t1: '4559A5', t2: '33589B', t3: '6F7DB3', m0: '33589B', m1: '5E71B3', en: '33589B', e1: '5E71B3',
```

Comment:
this looks like those weren't mapped

Reply:
Oh, that's really weird ... 
1st: isn't it supposed to be in a "color" object ?
2nd: Line 61 - The colors are correctly parsed
But the 2 fields are not at the same place : "pricol" is in the "amp" object, and "colors" is in the root object (by root, I mean the main object, with no name)
In the operatorAmp module, the colors are supposed to be correctly parsed, so I really don't know where this "pricol" come from :/

Reply:
i dunno either, lol. might check the static data for it in the file, it could have some incorrect copied data.

Reply:
Maybe, but I checked with a breakpoint the amp object in the constructor, and I didn't see any duplicated data

Reply:
Ahh, i see what's happening. It's because of [line 9 in the constructor](https://github.com/WFCD/arsenal-parser/blob/main/src/OperatorAmp.js#L9). I've got it falling back to the original data if nothing can be found on lookup from warframe-items, which makes sense. This looks good, in that case!